src/V3Randomize.cpp

```diff
@@ -1272,6 +1272,25 @@ class RandomizeVisitor final : public VNVisitor {
                 }
             }
             return stmtsp;
+        } else if (const auto* const unionDtp = VN_CAST(memberp ? memberp->subDTypep()->skipRefp()
+                                                                : exprp->dtypep()->skipRefp(),
+                                                        UnionDType)) {
+            AstNodeStmt* stmtsp = nullptr;
+            if (!unionDtp->packed()) {
+                unionDtp->v3warn(E_UNSUPPORTED, "Unsupported: IEEE 1800-2023: Unpacked unions "
+                                                "shall not be declared as rand or randc.");
```

Comment:
Need a "_bad.v"/"_bad.pl"/"bad.out" test for this - copy one of the others please.